### PR TITLE
Upgrade CircleCI to v2

### PR DESCRIPTION
**Pull request by guzzilar**
### 1. Objective
CircleCI now supported for multiple container. This feature allows us, to be able to test Omise-PHP in different environments.

This pull request is aiming to add more machine to test Omise-PHP in a various environment.

### 2. Description of change
- Upgrade CircleCI script, using version 2 

### 3. Quality assurance
All test scripts should work as normal.
Check: https://circleci.com/gh/omise/omise-php

### 4. Impact of the change

There is no container for PHP below v5.6.
(but I'm planing to drop support v5.3 anyway)

### 5. Additional notes

No

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      - run: php -r "if (hash_file('SHA384', 'composer-setup.php') === trim(file_get_contents('https://composer.github.io/installer.sig'))) { echo 'Installer verified'; } else { echo 'Installer invalid'; unlink('composer-setup.php'); } echo PHP_EOL;"
+      - run: php composer-setup.php
+      - run: php -r "unlink('composer-setup.php');"
+      - run: php ./composer.phar install
+      - run: php ./vendor/bin/phpunit tests
+
+  php56:
+    docker:
+      - image: circleci/php:5.6.38-cli-jessie-node-browsers
+    steps:
```
Comment (jonrandy) [inline .circleci/config.yml]: I don't know much about it, but is it possible to move these `steps` into something like a macro/function?

Comment (guzzilar) [reply .circleci/config.yml]: @jonrandy uhmmm I cannot find that kind of feature on their document.

Comment (jonrandy) [reply .circleci/config.yml]: Google is your friend... https://discuss.circleci.com/t/circle-yml-macros-includes/13721/4

Comment (guzzilar) [reply .circleci/config.yml]: lemmer try

Comment (jonrandy) [reply .circleci/config.yml]: It worked?

**Comment by guzzilar** [comment]
@jonrandy worked now. Thanks!

**Pull request merged**